src/test/java/com/google/graphgeckos/dashboard/api/DashboardControllerTest.java

```diff
@@ -0,0 +1,167 @@
+package com.google.graphgeckos.dashboard.api;
+
+import static org.mockito.BDDMockito.given;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.content;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.jsonPath;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.status;
+
+import com.google.cloud.Timestamp;
+import com.google.graphgeckos.dashboard.datatypes.*;
+import com.google.graphgeckos.dashboard.storage.DatastoreRepository;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.MockitoAnnotations;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.autoconfigure.web.servlet.AutoConfigureMockMvc;
+import org.springframework.boot.test.autoconfigure.web.servlet.WebMvcTest;
+import org.springframework.boot.test.mock.mockito.MockBean;
+import org.springframework.http.MediaType;
+import org.springframework.test.context.junit4.SpringRunner;
+import org.springframework.test.web.servlet.MockMvc;
+import org.springframework.test.web.servlet.ResultActions;
+import org.springframework.test.web.servlet.request.MockMvcRequestBuilders;
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.List;
+
+@RunWith(SpringRunner.class)
+@WebMvcTest(controllers = DashboardController.class)
+@AutoConfigureMockMvc
+public class DashboardControllerTest {
+
+  @MockBean
+  private DatastoreRepository datastoreRepository;
+
+  @Autowired
+  private MockMvc mvc;
+
+  private final String COMMIT_HASH = "1234";
+  private final Timestamp TIMESTAMP = Timestamp.now();
```

Comment:
I'd suggest creating the timestamp for a predefined time (e.g. .ofTimeMicroseconds). Even if this wouldn't affect the logic of the test, it would be a good practice to have tests that use the same parameters everytime.

Reply:
Done!